### PR TITLE
Added (Edit/Update User (JDBC, SQL)) into homework_10_proj_101222

### PR DESCRIPTION
**Pull request by kate-lvlup**

### CODE REVIEW

File: homework_10_proj_101222/src/main/java/app/RegistrationApp.java [hunk 1]
```diff
@@ -118,6 +114,13 @@ public void runApp() {
                         }
                         System.out.println();
                     }
+                    case 12 -> {
+                        User userBeforeEdit = userUtils.verifyUsername(bufferedReader);
+                        User userAfterEdit = new User();
+                        userUtils.fillUserFieldsForEditing(bufferedReader, userAfterEdit, userBeforeEdit);
+                        userService.saveUserAfterEditing(userAfterEdit);
```
Comment (YeaaaaH) [inline homework_10_proj_101222/src/main/java/app/RegistrationApp.java:121]: rename

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
fixs after lesson

File: homework_10_proj_101222/src/main/java/app/RegistrationApp.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_10_proj_101222/src/main/java/app/RegistrationApp.java:121]: saveUserAfterEditing() was deleted. New logic was applied.

**Comment by kate-lvlup** [comment]
fixs after lesson was added

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**